tracer/test/snapshots/Security.AspNetCore5AutoUserEvents.SecurityOn.defaultmode.TestUserLoginEvent_eventName=login.auto.success_bodyString=Input.UserName=TestUser&Input.Password=test.verified.txt

```diff
@@ -24,7 +24,7 @@
       runtime-id: Guid_1,
       span.kind: server,
       usr.id: Guid_2,
-      _dd.appsec.events.users.login.success.auto.mode: safe,
+      _dd.appsec.events.users.login.success.auto.mode: ident,
```

Comment:
Shouldn't safe get translated into anon?

Reply:
This file tests the default mode, the previous default was "safe", it's now "ident".

When "safe" is explicitly set, it should be translated to "anon". We verify that happens in the unit test, there were no explicit integration tests for safe mode, so I decided not add them. I think the unit tests are enough.